app/src/main/kotlin/com/wire/android/ui/theme/Color.kt

```diff
@@ -0,0 +1,25 @@
+package com.wire.android.ui.theme
+
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Stable
+import androidx.compose.ui.graphics.Color
+
+val WireLightColors = lightColors(
+    secondary = WireColor.LightBlue,
+    onSecondary = Color.White,
+    background =  WireColor.LightGray
+)
+
+val WireDarkColors = darkColors(
+
+)
+
+
+object WireColor {
```

Comment:
Bringing Compose (and UI to `.kt` files in general) raises a lot of MagicNumber warnings 🤦🏼‍♂️.

I guess adding `@Suppress("MagicNumber")` works for now. But it's good to keep in mind we might need to find a more sustainable way to solve this.